core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -791,52 +797,52 @@ class GroupMetadataManager(brokerId: Int,
         (removedOffsets, group.is(Dead), group.generationId)
       }
 
-    val offsetsPartition = partitionFor(groupId)
-    val appendPartition = new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, offsetsPartition)
-    getMagic(offsetsPartition) match {
-      case Some(magicValue) =>
-        // We always use CREATE_TIME, like the producer. The conversion to LOG_APPEND_TIME (if necessary) happens automatically.
-        val timestampType = TimestampType.CREATE_TIME
-        val timestamp = time.milliseconds()
+      val offsetsPartition = partitionFor(groupId)
```

Comment:
I fixed the indentation here while exploring mocks in a different way of testing this feature. I think it'd be good to commit this as it is confusing to read

Reply:
make senses to me. Could you fix the conflicting files?

Reply:
Will do!

Reply:
@stanislavkozlovski Any update? This part is worth being a separate PR :)